server/backends/blobstore/blobstore.go

```diff
@@ -265,7 +265,16 @@ func (d *DiskBlobStore) DeleteBlob(ctx context.Context, blobName string) error {
 	}
 	start := time.Now()
 	ctx, spn := tracing.StartSpan(ctx)
-	err = disk.DeleteFile(ctx, fullPath)
+
+	fileInfo, err := os.Stat(blobName)
+	if err != nil {
```

Comment:
should we move the `IsNotExist` check up into this `err != nil` block ? since before we would ignore IsNotExist errs

Reply:
Done.